docs/deploying/database.md

```diff
@@ -31,3 +31,88 @@ Migrations are [run automatically](https://github.com/elifesciences/elife-xpub-f
 Migrations have to be backward compatible as every deploy has a critical section where the old application code is still running after migrations have been applied.
 
 There is no automated rollback of migrations.
+
+## Backup and restore
+
+RDS snapshots are taken daily and [retained for 4 weeks in `prod`](https://github.com/elifesciences/builder/blob/master/projects/elife.yaml#L1970).
+
+The scripts `dump-database.sh` and `restore-database.sh`, on PATH in the servers, can respectively be used to perform a new dump from any Postgres host and to restore that dump.
```

Comment:
Where are these scripts? Would be good to add their location here.

Reply:
they are on the `PATH` so they should be accessible wherever you are, but I'll reword

Reply:
(they are in `/usr/local/bin` but you shouldn't have to know that to use them)